src/main/java/mServer/crawler/sender/MediathekSrf.java

```diff
@@ -168,6 +168,7 @@ private void addFilmsFromPage(MSStringBuilder page, String thema, String themePa
          * @param theme the theme name of the film
          */
         private void addFilms(String jsonMovieUrl, String themePageUrl, String theme) {
+            final String INDEX_0 = "index_0_av.m3u8";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)